### PR TITLE
Spanish Translation - Privacy chapter

### PR DESCRIPTION
**Pull request by decrecementofeliz**
Make progress on #498

### CODE REVIEW

File: src/config/2022.json [hunk 1]
```diff
@@ -1008,6 +1008,16 @@
...
+    "decrecementofeliz": {
+      "name": "Pilar Mera",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "1857745",
+      "github": "decrecementofeliz",
+      "twitter": "DecreceFeliz",
+      "website": "http://pi-comunicacion.com"
+    },
```
Comment (tunetheweb) [inline src/config/2022.json:1020]: @decrecementofeliz can you check you're happy with this?

**Review by tunetheweb — APPROVED** [review]
LGTM! Great effort.

One you confirm you're happy with your profile info, then good to merge

File: src/config/2022.json (hunk 1 above)
Comment (decrecementofeliz) [reply src/config/2022.json:1020]: Hi, yes!

Comment (tunetheweb) [reply src/config/2022.json:1020]: Sounds great! Always happy to get more translations.

**Pull request merged**